homeassistant/components/binary_sensor/services.yaml

```diff
@@ -0,0 +1,15 @@
+wink_pair_new_device:
```

Comment:
This should not be part of the binary sensor but instead be a service by the Wink component?

Reply:
I guess I could move it over there, but I was kinda basing this on how locks work. This service has to be called on the hub binary sensor.

Reply:
The binary sensor is only representing some state  of the hub. It's not the hub itself. The hub is a central part of the Wink system and thus this service should live on the wink component.